embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Breadcrumbs.kt

```diff
@@ -25,11 +25,5 @@ internal data class Breadcrumbs(
      * List of RN Action breadcrumbs.
      */
     @Json(name = "rna")
-    val rnActionBreadcrumbs: List<RnActionBreadcrumb>? = null,
-
-    /**
-     * List of captured push notifications
-     */
-    @Json(name = "pn")
-    val pushNotifications: List<PushNotificationBreadcrumb>? = null,
```

Comment:
Can the `LegacyPushNotificationDataSource` be deleted if we're getting rid of payload fields in the same changeset?